tests/common/db/malware.py

```diff
@@ -39,3 +43,14 @@ class Meta:
     created = factory.fuzzy.FuzzyNaiveDateTime(
         datetime.datetime.utcnow() - datetime.timedelta(days=7)
     )
+
+
+class MalwareVerdictFactory(WarehouseFactory):
+    class Meta:
+        model = MalwareVerdict
+
+    check = factory.SubFactory(MalwareCheckFactory)
+    release_file = factory.SubFactory(FileFactory)
+    classification = factory.fuzzy.FuzzyChoice([e for e in VerdictClassification])
```

Comment:
Looked this up out of curiosity and thought I'd share: `list(VerdictClassification)` _should_ also work here, although I see no reason for a change 😄

Reply:
Ooo yeah I like that. Let me see if I can change it everywhere.